datahub-web-react/src/app/entity/dataset/DatasetEntity.tsx

```diff
@@ -285,6 +285,8 @@ export class DatasetEntity implements Entity<Dataset> {
                 externalUrl={data.properties?.externalUrl}
                 statsSummary={data.statsSummary}
                 rowCount={(data as any).lastProfile?.length && (data as any).lastProfile[0].rowCount}
+                columnCount={(data as any).lastProfile?.length && (data as any).lastProfile[0].columnCount}
+                sizeInBytes={(data as any).lastProfile?.length && (data as any).lastProfile[0].sizeInBytes}
```

Comment:
not for now, but in the future if we keep adding fields from the lastProfile object here, we can just pass in that object instead of each field individually (reduce number of props and gives us more type safety)

Reply:
good idea